src/tactic/ui/panel/base_table_layout_wdg.py

```diff
@@ -9,6 +9,9 @@
 #
 #
 #
+
+import tacticenv
```

Comment:
usually, this would be present if there was a syntax error that did show up in TACTIC (something suppressed it).  This allows you to see it by being able to run the class from the command line.  It is now removed.